server/index.js

```diff
@@ -13,6 +13,9 @@ app.use(bodyParser.json());
 
 app.get('/product/:productId/item', (req, res) => {
   let item = req.params.productId;
+  res.header('Access-Control-Allow-Origin', '*');
```

Comment:
this line is allowing CORS from any source (`*`).  This means you're vulnerable to attack, it's probably ok for now, but we'll want to adjust this later when we get the proxy going, to only allow CORS from the proxy server.